src/Cli/dotnet/commands/dotnet-workload/install/WorkloadManifestUpdater.cs

```diff
@@ -161,37 +158,26 @@ public static void AdvertiseWorkloadUpdates()
             }
         }
 
-        public IEnumerable<(
-            ManifestVersionUpdate manifestUpdate,
-            Dictionary<WorkloadId, WorkloadDefinition> Workloads
-            )>
-            CalculateManifestUpdates()
+        public IEnumerable<ManifestUpdateWithWorkloads> CalculateManifestUpdates()
         {
-            var manifestUpdates =
-                new List<(ManifestVersionUpdate manifestUpdate,
-                    Dictionary<WorkloadId, WorkloadDefinition> Workloads)>();
             var currentManifestIds = GetInstalledManifestIds();
             foreach (var manifestId in currentManifestIds)
             {
-                var currentManifestVersion = GetInstalledManifestVersion(manifestId);
-                var advertisingManifestVersionAndWorkloads = GetAdvertisingManifestVersionAndWorkloads(manifestId);
-                if (advertisingManifestVersionAndWorkloads == null)
+                var (installedVersion, installedBand) = GetInstalledManifestVersion(manifestId);
```

Comment:
Ah, good find. I did this change like a few hours before I did the other deconstruction below. So, I didn't relate the new variable names to each other. Let me do that.

Reply:
Ah wait. I ended up looking at the method holistically. 

![image](https://github.com/dotnet/sdk/assets/17788297/c8e8f917-4cc4-4596-9b05-67327f809369)

The purpose of the method is to compare manifest versions (that's what `CalculateManifestUpdates` seems to mean). You can see the name matching I did between the underlined variables. So, I think keeping the name as since makes sense since they're all operating on manifests. Plus, if you hover over `installedVersion`, it says the type is `ManifestVersion`. Further clarification shouldn't be necessary when actually interacting with the code, but it is definitely harder to see in a code review.

Reply:
After seeing the screenshot above when I sat down at my desk, I realized I could move the variable creation together. Now, `installedVersion`/`installedBand` are created right before `adVersion`/`adBand`.